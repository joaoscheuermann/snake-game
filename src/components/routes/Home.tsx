import { Link } from "react-router";
import { Button, Flex, Heading, Text } from "@radix-ui/themes";

export default function Home() {
  return (
    <Flex
      direction="column"
      align="center"
      justify="center"
      className="h-screen"
      gap="9"
    >
      <Heading size="9" className="mb-8">
        Snake!
      </Heading>
      <Text>Click in the Start Game button to start playing</Text>
      <Button asChild size="3" variant="surface">
        <Link to="/play">Start Game</Link>
      </Button>
    </Flex>
  );
}
