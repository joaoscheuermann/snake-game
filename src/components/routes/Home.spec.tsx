import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import Home from "./Home";

describe("Home", () => {
  it("should render correctly", () => {
    const { asFragment } = render(
      <MemoryRouter>
        <Home />
      </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
