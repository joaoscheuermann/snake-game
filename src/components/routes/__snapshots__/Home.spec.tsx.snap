// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly 1`] = `
<DocumentFragment>
  <div
    class="rt-Flex rt-r-fd-column rt-r-ai-center rt-r-jc-center rt-r-gap-9 h-screen"
  >
    <h1
      class="rt-Heading rt-r-size-9 mb-8"
    >
      Snake!
    </h1>
    <span
      class="rt-Text"
    >
      Click in the Start Game button to start playing
    </span>
    <a
      class="rt-reset rt-BaseButton rt-r-size-3 rt-variant-surface rt-Button"
      data-accent-color=""
      data-discover="true"
      href="/play"
    >
      Start Game
    </a>
  </div>
</DocumentFragment>
`;
